rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -129,6 +135,15 @@ local function sync_backend(backend)
 end
 
 local function sync_backends()
+  local timestamp = configuration.get_timestamp_data()
+  if timestamp <= last_timestamp then
```

Comment:
Should we force sync if `last_timestamp` - `timestamp` is too large?

Reply:
Do you mean we should force sync if **current timestamp** is too larger than ``backends_last_sycned_at`` stored in each worker, avoiding failure of backends syncing caused by error updating ``raw_backends_last_sycned_at`` in shared configuration data?